custom/scripts/eks/litmus-kafka-deployer/terraform.tfstate

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Does the user needs this file, generally it contains the state of created infrastructure?